Sources/protoc-gen-swift/MessageFieldGenerator.swift

```diff
@@ -52,6 +52,28 @@ extension Google_Protobuf_FieldDescriptorProto {
         let m = context.getMessageForPath(path: typeName)!
         return m.options.mapEntry
     }
+    // Returns true/false if the field will be hold a Message. map<> is the
+    func getFieldHoldsMessage(context: Context) -> Bool {
+        switch label {
+        case .required:
+            fallthrough
+        case .optional:
+            return type == .message || type == .group
+        case .repeated:
+            if type == .group { return true }
```

Comment:
The indentation in this block looks like a combination of 2 and 4 spaces, making it a little hard to follow.